blocks/gmo-campaign-details/gmo-campaign-details.js

```diff
@@ -0,0 +1,526 @@
+        'channel': 'Adobe.com',
+        'reviewLinkName': 'Review Link',
+        'reviewLinkHref': '#',
+        'finalAssetName': 'Final Asset',
+        'finalAssetHref': '#',
+        'kpi': 'KPI',
+        'statusUpdate': '57'
+    },
+    {
+        'category': 'Use',
+        'subcategory': 'Customer Story',
+        'name': 'Maegan',
+        'type': 'Animated Display',
+        'channel': 'Display',
+        'reviewLinkName': 'Review Link',
+        'reviewLinkHref': '#',
+        'finalAssetName': 'Final Asset',
+        'finalAssetHref': '#',
+        'kpi': 'KPI',
+        'statusUpdate': '17'
+    }
+];
+
+export default async function decorate(block) {
+    const rows = buildRows(testData);
+    block.innerHTML = `
+    <div class="back-button">
+        <span class="icon icon-back"></span>
+        <span class="back-label">Back</span>
+    </div>
+    <div class="main-body-wrapper">
+        <div class="details-header-wrapper">
+            <div class="campaign-img">
+            </div>
+            <div class="header-title">
+                <div class="header-row1">
+                    <span class="h1">Express Mobile Beta</span>
+                    <div class="campaign-status">In Progress</div>
+                </div>
+                <div class="header-row2">
+                    <span class="icon icon-calendar"></span>
+                    <span class="campaign-date">03/07/2024</span>
+                </div>
+            </div>
+        </div>
+        <div class="tab-wrapper">
+            <div id="tab1toggle" class="tab active">Overview</div>
+            <div id="tab2toggle" class="tab">Deliverables</div>
+            <div id="tab3toggle" class="tab ">Calendar</div>
+        </div>
+        <div id="tab1" class="two-column overview inactive">
+            <div class="overview-wrapper">
+                <span class="h1 overview-heading">At a Glance</span>
+                <span class="h3">Strategy</span>
+                <span class="description">
+                    Express mobile public beta is not a major at scale marketing moment (due to the limited nature of beta experience) with key audiences of
+                    Existing Express users, investors and media. Marketing approach is signaling to the market our continued momentum with the new mobile
+                    beta release, focusing efforts on PR, social/community and in-app surfaces.
+                </span>
+                <div class="button no-bg">Read more</div>
+                <div class="kpis-wrapper">
+                    <span class="h3">KPIs to Measure Success</span>
+                    <ul>
+                        <li>PR impressions & dedicated earned stories</li>
+                        <li>Mobile exports</li>
+                        <li>Community & social interactions</li>
+                        <li>100% by EOL</li>
+                    </ul>
+                </div>
+                <div class="use-cases-wrapper">
```

Comment:
Comment out "Hero use case" and "Links to Important Artifacts" for MVP

Reply:
Is it okay if I hide them with CSS?